src/app/manage/user-details/user-details.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { UserService } from 'src/app/user.service';
+import { trigger, state, style, transition, animate } from '@angular/animations';
+import { UserStatusPipe } from '../user-status.pipe';
+
+@Component({
+  selector: 'app-user-details',
+  templateUrl: './user-details.component.html',
+  styleUrls: ['./user-details.component.css'],
+  animations:[
+    trigger('userDetailAnimate',[ 
+      state('open', style({
+        opacity: 0,
+      })),
+      state('closed', style({
+        opacity: 0.5,
+      })),
+      transition('open <=> closed', [
+        animate('1s')
+      ]),
+    ])
+  ],
+  providers:[UserStatusPipe]
+})
+export class UserDetailsComponent implements OnInit {
+
+  user: any;
```

Comment:
never use any. Create a proper interface and assign the type. To avoid undefined error, have a default initialization.